README.md

```diff
@@ -37,7 +37,7 @@ The number of `#` characters needed for the section titles is dynamically calcul
 
 ## Requirements
 
-The project has been developed and tested with node version `12.21.0`.
+The project has been developed and tested with node version `16.14.2`.
```

Comment:
Since we use the Node version from GitHub and Docker, I would simply say `16.x`.